- No `T' option available in OpenBSD ln(1).
- OpenBSD has locale_t but not the strtod_l() family.

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -86,7 +86,8 @@ if cc.has_header_symbol('unistd.h', 'issetugid')
     config_h.set('HAVE_ISSETUGID', 1)
 endif
 
-if cc.has_header_symbol('locale.h', 'locale_t', prefix : '#define _GNU_SOURCE')
+if cc.has_header_symbol('locale.h', 'locale_t', prefix : '#define _GNU_SOURCE') and \
+   cc.has_function('strtod_l', prefix : '#define _GNU_SOURCE\n#include <stdlib.h>')
     config_h.set('HAVE_LOCALE_T', 1)
 endif
 
@@ -278,7 +279,7 @@ endif
 # Create a symlink from the build root to the source root. This is used when
 # running libcamera from the build directory to locate resources in the source
 # directory (such as IPA configuration files).
-run_command('ln', '-fsT', meson.project_source_root(), meson.project_build_root() / 'source',
+run_command('ln', '-fs', meson.project_source_root(), meson.project_build_root() / 'source',
             check : true)
 
 configure_file(output : 'config.h', configuration : config_h)
